spring-boot-app-example/src/test/java/io/github/mfvanek/pg/cluster/exceptions/ExceptionHandlerTest.java

```diff
@@ -0,0 +1,70 @@
+package io.github.mfvanek.pg.cluster.exceptions;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.FieldError;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.NoSuchElementException;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.mockito.Mockito.*;
+
+public class ExceptionHandlerTest {
```

Comment:
We need a kind of integration test here.
For example, make a request via webTestClient and got `handleInvalidUUIDFormat` error.

Reply:
Done it